docker-compose.yml

```diff
@@ -1,13 +1,5 @@
 services:
 
-#  janus:
```

Comment:
What component was nginx supporting again? I'm assuming removal won't affect Docker performance?

Reply:
Wasn't using anything, it was something we tried to help resolve docker issues but it's not needed.